pkg/svcat/service-catalog/class_test.go

```diff
@@ -51,7 +51,7 @@ var _ = Describe("Class", func() {
 
 	Describe("RetrieveClasses", func() {
 		It("Calls the generated v1beta1 List method", func() {
-			classes, err := sdk.RetrieveClasses()
+			classes, err := sdk.RetrieveClasses(nil)
```

Comment:
This is going to cause a panic because `RetrieveClasses` is not expecting to be passed nil. You need to either check for nil in `RetrieveClasses` or pass in a `&FilterOptions{}` instead.

Reply:
I'd be ok with either making the opts parameter a struct, not a pointer, in the `RetrieveClasses` method signature and then always passing in an empty struct. Or checking at the top of that method for a nil opts, and then defaulting it to an empty struct.